docs/_docs/Developers/build.md

```diff
@@ -62,6 +94,8 @@ cd phoenicis
 mvn clean package
 ```
 
+---
+
 ## Fedora >= 29
```

Comment:
Is there a reason why the section for Arch Linux, Fedora and TrueOS are not changed the same way as for Ubuntu and Gentoo?

Reply:
@madoar i don't know the instructions for arch and trueOS.

Reply:
I think we should have a common style across all OS sections.
My comment is not really about the content just the style of the sections.

Reply:
*confused*

Try to make a commit i guess?